frontend/src/components/App/App.css

```diff
@@ -30,6 +30,11 @@ h2 {
   text-transform: uppercase;
 }
 
+h4 {
```

Comment:
So this reduces the spacing too? :curious-parrot:

Reply:
no, no code change in for line spacing